### ISSUE TITLE
Scalar writing frequency too high will lead to data loss?

### ISSUE THREAD

**Issue by chivee** [lanpa/tensorboardX#201]
Hi @lanpa ,

Thanks for your great work. I really enjoy the tensorboardx journey.
However,  the data loss issues are haunting me, which was that if the Summary writer writing too many scalars. there will be a data loss issue.

do you have any comment on this?

**Comment by lanpa** [comment]
I just saw a bug report on missing `text` event. What did you mean by `too many` and how did you spot the missing value? It will be good to provide some code for me to debug.

**Comment by chivee** [comment]
![image](https://user-images.githubusercontent.com/2283057/43403238-268cee2a-9447-11e8-8a42-ee4160730357.png)


```
import torch
import random
import torchvision.models as models
from tensorboardX import SummaryWriter

writer = SummaryWriter()
writer_list = []

for i in range(10):
    writer_list.append(SummaryWriter('test/{0}'.format(i)))

writer_item = []
for i in range(10):
    writer_item.append("data/test_name_{0}".format(i))

for n_iter in range(100000):
    for i in range(10):
        writer = writer_list[i]
        for j in range(10):
            rnd_value = random.uniform(0.9,1.1)
            writer.add_scalar('{0}'.format(writer_item[j]), rnd_value, n_iter)

# export scalar data to JSON for external processing
writer.export_scalars_to_json("./all_scalars.json")
writer.close()
```

**Comment by lanpa** [comment]
I didn't run the code, but I guess the data points should be there. see https://github.com/lanpa/tensorboardX/issues/44

**Comment by lanpa** [comment]
<img width="927" alt="Screen Shot 2019-07-17 at 10 54 34 PM" src="https://user-images.githubusercontent.com/2005323/61385919-e1284280-a8e5-11e9-8533-f0bf46298695.png">
Confirms that it should working and the size of event file is 53M per writer, which implies 53B per each record, reasonable.
